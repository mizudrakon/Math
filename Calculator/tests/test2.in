6
a =3
b=$a
A= 3
b= $a * 5
c=4
$a * $B * $C