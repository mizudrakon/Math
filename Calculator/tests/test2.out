invalid
invalid
3
15
4
180